paddle/framework/data_transform.cc

```diff
@@ -111,5 +112,22 @@ void TransDataLayout(const platform::DeviceContext* ctx,
 }  // namespace paddle
```

Comment:
Parameter order should be inputs then outputs: line 106 `CastDataLayout(src, dst, ctx, axis)` -> `CastDataLayout( ctx, axis,src, dst)`

Reply:
done.